NEWS.md

```diff
@@ -1,5 +1,13 @@
 ## 3.1.0 - Unreleased
 
+## 3.0.2 - Released (Quesnelia R1 2024)
+The primary focus of this release was updating vertx dependency
+[Full Changelog](https://github.com/folio-org/edge-orders/compare/v3.0.1...v3.0.2)
+
+### Dependencies
+* Bump `vertx` from `4.5.4` to `4.5.7`
```

Comment:
better to specify also ticket number to have more context why versions bump is needed

Reply:
Thanks, forgot comment has ticket. Added